stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1234,6 +1249,11 @@ public final class TestSuite {
     case .parameterized(code: let code, _):
       code(parameter!)
     }
+
+#if SWIFT_RUNTIME_ENABLE_LEAK_CHECKER
```

Comment:
@CodaFi 

@gribozavr is correct. You need this:

--- a/stdlib/private/StdlibUnittest/CMakeLists.txt
+++ b/stdlib/private/StdlibUnittest/CMakeLists.txt
@@ -5,6 +5,10 @@ set(swift_stdlib_unittest_framework_depends)
 set(swift_stdlib_unittest_compile_flags
   "-Xfrontend" "-disable-objc-attr-requires-foundation-module")
 
+if (SWIFT_RUNTIME_ENABLE_LEAK_CHECKER)
+  list(APPEND swift_stdlib_unittest_compile_flags "-DSWIFT_RUNTIME_ENABLE_LEAK_CHECKER")
+endif()
+
 if(SWIFT_HOST_VARIANT MATCHES "${SWIFT_DARWIN_VARIANTS}")
   list(APPEND swift_stdlib_unittest_platform_sources
     GetOSVersion.mm)